cmd/webhook/app/options/options.go

```diff
@@ -70,7 +70,7 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 		"The directory that contains the server key and certificate.")
 	flags.StringVar(&o.CertName, "tls-cert-file-name", "tls.crt", "The name of server certificate.")
 	flags.StringVar(&o.KeyName, "tls-private-key-file-name", "tls.key", "The name of server key.")
-	flags.StringVar(&o.TLSMinVersion, "tls-min-version", defaultTLSMinVersion, "Minimum TLS version supported. Possible values: 1.0, 1.1, 1.2, 1.3.")
+	flags.Float32Var(&o.TLSMinVersion, "tls-min-version", defaultTLSMinVersion, "Minimum TLS version supported. Possible values: 1.0, 1.1, 1.2, 1.3.")
```

Comment:
I'm not sure if changing the flag type would break the backward compatibility, what do you think?